node/src/proxy_client/stream_handler_pool.rs

```diff
@@ -194,6 +195,7 @@ impl StreamHandlerPoolReal {
                     Some(wallet) => inner
                         .accountant_sub
                         .try_send(ReportExitServiceProvidedMessage {
+                            timestamp: SystemTime::UNIX_EPOCH, // TODO: Drive this in
```

Comment:
? if you think it still need a proof, maybe it's missing then... or remove the comment...